common/workunit/workflow.cpp

```diff
@@ -383,6 +386,7 @@ class CCloneWorkflowItem : public CInterface, implements IRuntimeWorkflowItem
         persistWfid = other->queryPersistWfid();
         scheduledWfid = other->queryScheduledWfid();
         persistCopies = other->queryPersistCopies();
+        persistRefresh = other->queryPersistRefresh();
         other->queryCluster(clusterName);
```

Comment:
Done with queryXX for consistency - all params are using queryXX. Refactor code to use getXX in another ticket later.